rest-tests/src/test/resources/features/common.feature

```diff
@@ -1,7 +1,32 @@
 Feature: Common scenarios
 
-@clean-deploy
-Scenario: Clean & deploy
-	Given clean default namespace
-	When deploy Syndesis from template
-	Then wait for Syndesis to become ready
\ No newline at end of file
+  @clean-deploy
+  Scenario: Clean & deploy
+    Given clean default namespace
+    When deploy Syndesis from template
+    Then wait for Syndesis to become ready
+
+  @integrations-twitter-to-salesforce
+  Scenario: Tw - sf integration
+    Given user cleans tw to sf scenario
+    And user creates the tw connection using "twitter_listen" template
+    And user creates sf connection
```

Comment:
I don't see this step implementation, is it possible? :)

Reply:
all of the steps are implemented somewhere, it builds with no problem :)

Reply:
added, srry my bad